### DISCUSSION TITLE
RA not running on Linux

### DISCUSSION THREAD

**Discussion by noau** [zed-industries/zed#14280] [Feature Requests]
I installed `zed` on my EndeavourOS(Arch) last day as it's released, and set ra inlay hints up in settings following the docs:
```json
  "inlay_hints": {
    // Global switch to toggle hints on and off, switched off by default.
    "enabled": true
  },
  "lsp": {
    "rust-analyzer": {
      "initialization_options": {
        "inlayHints": {
          "maxLength": null,
          "lifetimeElisionHints": {
            "useParameterNames": true,
            "enable": "skip_trivial"
          },
          "closureReturnTypeHints": {
            "enable": "always"
          }
        }
      }
    }
  }
```
But when I opened an Rust project and edit `.rs` files, it doesn't work. There's only basic highlights without any other features, and I cheked language server logs(`debug: open language server logs`), there's no RA running. 

So is there anything missed or wrong in me setup?

**Comment by SomeoneToIgnore** [comment]
Looks correct, so maybe rust-analyzer download or startup had failed?

Can you re-open Zed in that project again and try editing something, then get `/home/user/.local/share/zed/logs/Zed.log` or run a `zed: open log` command inside Zed and get that contents pasted here?

**Reply by noau** [reply]
Thanks for your help! In the log I found that it reports some certificate errors:
```log
2024-07-12T17:44:23+08:00 [ERROR] Client(Error { kind: BadServerCertificate, context: None, source: Some(Error { description: "SSL peer certificate or SSH remote key was not OK", code: 60, extra: None }), source_type: Some("curl::error::Error"), local_addr: Some(127.0.0.1:51096), remote_addr: Some(127.0.0.1:443) })
```
so I switched to `"path_lookup": true` for using my local ra installation.

I also tried setting `SSL_CERT_FILE` to my certificate, but it doesn;t work. Anyway, thanks for your help :)

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
